web/core/src/jsMain/kotlin/org/jetbrains/compose/web/events/SyntheticTouchEvent.kt

```diff
@@ -0,0 +1,18 @@
+package org.jetbrains.compose.web.events
+
+import androidx.compose.web.events.SyntheticEvent
+import org.w3c.dom.TouchEvent
+import org.w3c.dom.TouchList
+import org.w3c.dom.events.EventTarget
+
+class SyntheticTouchEvent(
+    nativeEvent: TouchEvent,
+) : SyntheticEvent<EventTarget>(nativeEvent) {
+
+    val altKey: Boolean = nativeEvent.altKey
+    val changedTouched: TouchList = nativeEvent.changedTouches
```

Comment:
type: I guess it was supposed to be called changeTouches

Reply:
fixed, thanks!